app/models/form_submission.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: form_submissions
+#
+#  id              :bigint           not null, primary key
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint           not null
+#  person_id       :bigint           not null
+#
+# Indexes
+#
+#  index_form_submissions_on_organization_id  (organization_id)
+#  index_form_submissions_on_person_id        (person_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (organization_id => organizations.id)
+#  fk_rails_...  (person_id => people.id)
+#
+class FormSubmission < ApplicationRecord
+belongs_to :person
+belongs_to :organization
```

Comment:
Check out other resources that belong to an organization (most of them) and you'll see we use the acts as tenant helper here. This is important for setting the tenancy, or scope, for a given record i.e., which org it belongs to, and is used by the acts as tenant scoping logic elsewhere.